src/types.js

```diff
@@ -92,6 +92,7 @@ export type ProjectInternal = {
     color: string,
     icon: string,
     createdAt: number,
+    isImported: boolean,
```

Comment:
This also appears unused, outside of the middleware (which has been replaced with a saga)